### DISCUSSION TITLE
Prometheus  Concourse Pipeline metrics details

### DISCUSSION THREAD

**Discussion by prime89-hash** [concourse/concourse#7949] [Help & Support]
Hi @taylorsilva,
Just need your help in one information.
Just wanted to check how we can add concourse Prometheus metric to cloudwatch dashboard is there any emitter available for the same

**Comment by taylorsilva** [comment]
Don't use cloudwatch, but guessing unless Cloudwatch can interface with one of the existing emitters, like Prometheus, that there isn't a way to get it working.
